proto/buf.md

```diff
@@ -1,17 +1,17 @@
 # Protobufs
 
-This is the public protocol buffers API for [Mesh Security SDK](https://github.com/osmosis-labs/mesh-security-sdk).
+This is the public protocol buffers API for [Babylon SDK](https://github.com/babylonchain/babylon-sdk).
 
 ## Download
 
 The `buf` CLI comes with an export command. Use `buf export -h` for details
 
 #### Examples:
 
-Download mesh-security protos for a commit:
+Download Babylon SDK protos for a commit:
```

Comment:
Hmm, perhaps at some point we'll need to start talking about the Babylon Consumer SDK.

But I think it's better / simpler without it for now.